src/screens/RatingsModal/DisappointedForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from "react";
+import { useSelector } from "react-redux";
+import useFeature from "@ledgerhq/live-common/lib/featureFlags/useFeature";
+import { Flex } from "@ledgerhq/native-ui";
+import { WebView } from "react-native-webview";
+import styled from "styled-components/native";
+import { track } from "../../analytics";
+import { ratingsHappyMomentSelector } from "../../reducers/ratings";
+
+const injectedJavascript = `
+setTimeout(function() {
```

Comment:
is it possible to implement a "wait for item to be there / retry logic" rather than a one shot strategy like this timeout of an arbitrary duration ? with a `setInterval` it should be pretty straightforward